openapi-generator/src/main/kotlin/org/jellyfin/openapi/Generator.kt

```diff
@@ -35,6 +35,9 @@ class Generator(
 		.map(apiBuilder::build)
 		.map(fileSpecBuilder::build)
 
+	private fun createApiConstants(info: io.swagger.v3.oas.models.info.Info): FileSpec = openApiConstantsBuilder.build(info)
```

Comment:
I suppose an import would be reasonable here. Could use an alias import is `Info` isn't unique enough.

Reply:
Android Studio somehow did it, fixed now.